src/SSRThread.cpp

```diff
@@ -45,7 +51,7 @@ void SSRThread::run()
     start_ssr_uv_local_server(profile);
 }
 
-SSRThread::~SSRThread()
+void SSRThread::stop()
```

Comment:
QThread does not have a restart mechanism. If you stop it manually, you will not be able to start it again. It may be a better idea to use destructor with unique_ptr